### PR TITLE
homework week3 databases

### PR DESCRIPTION
**Pull request by 1620prabn**
all files have been added

### CODE REVIEW

**Review by babakbandpey — CHANGES_REQUESTED** [review]
Hello Prabin

The create statements are not included. Those are all needed for me to be able to test your queries.
Please add them.
Thank you

**Comment by 1620prabn** [comment]
I have already added all the files. Do you want it to be in one file?

**Comment by 1620prabn** [comment]
On Mon, 24 Aug 2020 at 11.47, Prabin Maharjan <1620prabin@gmail.com> wrote:

> I have already added all the files.
>

https://github.com/1620prabn/hyf-homework/blob/33028b43b3364156f946edab15363b2c2d27c89d/databases/week3/meal_data.sql


Do you mean this one? Its already on github

File: databases/week3/additional queries.sql [hunk 1]
```diff
@@ -0,0 +1,35 @@
+-- Get meals that has a price smaller than a specific price fx 90 --
+SELECT * FROM meal 
+WHERE price < 700;
+
+-- Get meals that still has available reservations--
+SELECT * FROM meal, reservation
+WHERE  meal.max_reservation > reservation.number_of_guest;
+
```
Comment (babakbandpey) [inline databases/week3/additional queries.sql:8]: There is a problem here.

Please read this: https://hackyourfuture-cph.slack.com/archives/C010GF9E2MU/p1597953828024800

File: databases/week3/additional queries.sql [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+SELECT * FROM meal 
+WHERE title LIKE '%classic%' ;
+
+-- Get meals that has been created between two dates-- 
+SELECT * FROM meal
+WHERE created_date BETWEEN '2020-08-19' AND '2020-08-21' ;
+
+-- Get only specific number of meals fx return only 5 meals--
+
+SELECT COUNT(id) FROM meal
```
Comment (babakbandpey) [inline databases/week3/additional queries.sql:20]: `select * from meals limit 5;`

File: databases/week3/additional queries.sql [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+limit 5 ;
+
+-- Get the meals that have good reviews --
+SELECT meal.title FROM meal, review
```
Comment (babakbandpey) [inline databases/week3/additional queries.sql:24]: You need the select the `avg` of stars given to a meal. It means that you need to group by `meal_id`

File: databases/week3/review.sql [hunk 4]
```diff
@@ -0,0 +1,30 @@
+INSERT INTO review (id, title, description, meal_id, stars, created_date, meal_id1)
+VALUES(1,'burger', 'very tasty burger',1, 5, '2020-08-19 02:03:20', 1 );
```
Comment (babakbandpey) [inline databases/week3/review.sql:2]: why isn't it a god idea to define the primary key when it is already auto increment?

**Review by babakbandpey — CHANGES_REQUESTED** [review]
Hello Prabin,

I have some comments.
Thanks

**Pull request merged**